x/sequencer/keeper/hook_listener.go

```diff
@@ -55,15 +74,20 @@ func (hook rollappHook) FraudSubmitted(ctx sdk.Context, rollappID string, height
 		return err
 	}
 
-	// unbond all other bonded sequencers
-	sequencers := hook.k.GetSequencersByRollappByStatus(ctx, rollappID, types.Bonded)
-	for _, sequencer := range sequencers {
-		err := hook.k.forceUnbondSequencer(ctx, sequencer.SequencerAddress)
+	// unbond all other sequencers
+	bonded := hook.k.GetSequencersByRollappByStatus(ctx, rollappID, types.Bonded)
+	unbonding := hook.k.GetSequencersByRollappByStatus(ctx, rollappID, types.Unbonding)
+	for _, sequencer := range append(bonded, unbonding...) {
+		err := hook.k.unbondSequencer(ctx, sequencer.SequencerAddress)
 		if err != nil {
 			return err
 		}
 	}
 
+	// clear the proposer and next proposer
+	hook.k.removeProposer(ctx, rollappID)
+	hook.k.removeNextProposer(ctx, rollappID)
```

Comment:
duplicated in slashing

Reply:
slashing is handling a specific sequencer
while here is at the rollapp level